integration/local-gadget/integration_test.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	containercollection "github.com/kinvolk/inspektor-gadget/pkg/container-collection"
+)
+
+const (
+	ContainerRuntimeDocker     = "docker"
+	ContainerRuntimeContainerd = "containerd"
+	ContainerRuntimeCRIO       = "cri-o"
+)
+
+var supportedContainerRuntimes = []string{ContainerRuntimeDocker, ContainerRuntimeContainerd, ContainerRuntimeCRIO}
+
+var (
+	integration      = flag.Bool("integration", false, "run integration tests")
+	containerRuntime = flag.String("container-runtime", "", "allows to skip tests that are not supported on a given container-runtime")
```

Comment:
hehe. Side effect of ctrl-c + ctrl-v :smile: But my initial idea was to do some validation like the docker version since we need specific one for cri-o